postgres/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:16.04
+ENV RUNTIME_PACKAGES="python3"
+ENV BUILD_PACKAGES="curl build-essential python3-dev ca-certificates libssl-dev libffi-dev postgresql libpq-dev"
+
+RUN apt-get update && apt-get install -y $RUNTIME_PACKAGES $BUILD_PACKAGES && curl -sS https://bootstrap.pypa.io/get-pip.py | python3
```

Comment:
Did you see the other comment I left? It seems to have dissapeared...

Basically - installing python3 and pip here might be extraneous, as it is there by default on python 3.4> (Ubuntu 16.04 has python 3.5.1 by default).